woocommerce/woocommerce
# [CYS] Update sidebar links copy

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/44700



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure the `WooCommerce Beta Tester` plugin is installed and activated (available on this monorepo).
2. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
3. Go to `WooCommerce > Home`, click on `Start customizing`, and finish the flow.
4. Make sure the sidebar section copy looks like 👇 (test on a local/JN installation and on WooExpress).

| Local/JN (Core) | WooExpress |
|--------|--------|
| <img width="385" alt="Screenshot 2024-02-19 at 10 45 26" src="https://github.com/woocommerce/woocommerce/assets/186112/d6dc9cd8-13f4-4c9f-b7c9-c82d36d90364"> | <img width="375" alt="Screenshot 2024-02-19 at 10 45 17" src="https://github.com/woocommerce/woocommerce/assets/186112/bff647e9-72b6-4fc1-b7c8-cf36fbf9aacb"> | 




### Changelog entry




<details>

#### Significance




#### Type




#### Message 
CYS - Update sidebar links copy on the assembled page.
#### Comment 

</details>
